openshift-examples/keycloak-https.json

```diff
@@ -207,6 +213,14 @@
                                     {
                                         "name": "DB_VENDOR",
                                         "value": "${DB_VENDOR}"
+                                    },
+                                    {
+                                        "name": "JGROUPS_DISCOVERY_PROTOCOL",
+                                        "value": "dns.DNS_PING"
+                                    },
+                                    {
+                                        "name": "JGROUPS_DISCOVERY_PROPERTIES",
+                                        "value": "${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local"
```

Comment:
This seems wrong, it should be `"value": "dns_query=${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local"`